source/installer.py

```diff
@@ -363,7 +363,11 @@ def tryRemoveFile(path,numRetries=6,retryInterval=0.5,rebootOK=False):
 		time.sleep(retryInterval)
 	if rebootOK:
 		log.debugWarning("Failed to delete file %s, marking for delete on reboot"%tempPath)
-		winKernel.moveFileEx("\\\\?\\"+tempPath,None,winKernel.MOVEFILE_DELAY_UNTIL_REBOOT)
+		try:
+			# #9847: convert the path qualifier to Unicode so temp path can be appended correctly.
+			winKernel.moveFileEx(u"\\\\?\\"+tempPath,None,winKernel.MOVEFILE_DELAY_UNTIL_REBOOT)

```

Comment:
no need for the github issue number, more helpful is "can't use a raw string, the trailing slash escapes the closing quote"

Personally I think that should be considered a bug in the python parser...

Reply:
I think this comment is still not really telling us anything. What are "certain issues"? I also don't think it has anything to do with appending a string, this is likely to mislead the reader. Try the following in a python 3 console:

In Python 3.7.3:
``` Python
>>> raw = r"\\?\"
  File "<stdin>", line 1
    raw = r"\\?\"
                ^
SyntaxError: EOL while scanning string literal
```

The "what" part of this comment is also quite misleading if you aren't part of the conversation we are having on this PR. The next line is moving a file, not converting a path.

I would change these two lines to:

``` Python
# Use escapes in a unicode string instead of raw. 
# In a raw string the trailing slash escapes the closing quote leading to a python syntax error.
pathQualifier=u"\\\\?\\"
# Move file to None to delete it
winKernel.moveFileEx(pathQualifier+tempPath,None,winKernel.MOVEFILE_DELAY_UNTIL_REBOOT)
```